src/main/webapp/app/exercises/shared/plagiarism/plagiarism-inspector/plagiarism-inspector.component.ts

```diff
@@ -0,0 +1,135 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { HttpResponse } from '@angular/common/http';
+import { ActivatedRoute, Router } from '@angular/router';
+import { Subject, Subscription } from 'rxjs';
+import { ModelingExerciseService, ModelingSubmissionComparisonDTO } from 'app/exercises/modeling/manage/modeling-exercise.service';
+import { ModelingExercise } from 'app/entities/modeling-exercise.model';
+import { downloadFile } from 'app/shared/util/download.util';
+import { ExportToCsv } from 'export-to-csv';
+
+@Component({
+    selector: 'jhi-plagiarism-inspector',
+    styleUrls: ['./plagiarism-inspector.component.scss'],
+    templateUrl: './plagiarism-inspector.component.html',
+})
+export class PlagiarismInspectorComponent implements OnDestroy, OnInit {
+    selectedComparisonIndex: number;
+    checkPlagiarismInProgress: boolean;
```

Comment:
Should this not be named something like `isPlagiarismCheckInProgress`?

Reply:
I don't exactly know about our preferred naming convention, but personally, I would use `isPlagiarismCheckInProgress()` as a function name (asking a question) and `plagiarismCheckInProgress` as the name of the answer to that question, containing the actual boolean value.

I could rename `checkPlagiarismInProgress` to `plagiarismCheckInProgress`, as the prefix `check` might be confusing and more suitable for a function name.